webbeans-tomcat7/pom.xml

```diff
@@ -105,6 +105,50 @@
         </dependency>
     </dependencies>
 
+    <build>
```

Comment:
the changes to this file could go in a separate PR, but they will speed up the build considerably, while also enabling running the integration tests in a lot of corporate network environments

Reply:
hmm, not locally due to "checks" (not a big deal but reporting it)